content/usgs-packages/dataRetrieval_readNWIS.Rmd

```diff
@@ -315,7 +330,7 @@ makeArgDefTable("readNWISqw", table.num=8)
 
 or_site_do <- readNWISqw(siteNumbers=c("455415119314601", "454554119121801"), parameterCd="00300",
                         startDate="2015-10-01", endDate="2016-09-30")
-head(or_site_do)
+head(or_site_do[,c("site_no","sample_dt","result_va")])
```

Comment:
could we add a `ncol(or_site_do)` and then a comment about looking at just a few columns?